src/layouts/EditHomepage/EditHomepage.jsx

```diff
@@ -1082,6 +1226,55 @@ const EditHomepage = ({ match }) => {
                                       />
                                     </Editable.DraggableAccordionItem>
                                   )}
+
+                                  {showNewLayouts &&
+                                    section.announcements &&
+                                    errors.sections[sectionIndex]
+                                      .announcements && (
```

Comment:
(can't see prev comment) - this was the hotfix to an issue you found previously right? could you just add a comment here explaining the issue + fix so that we have clarity going forward

Reply:
ef614d16eb6a7c4eed4d74329c310e5f4a00cb69